accounts/abi/bind/precompilebind/precompile_bind_test.go

```diff
@@ -467,7 +467,7 @@ var bindTests = []struct {
 			require.Equal(t, testGreeting, unpackedGreeting)
 
 			// test that the allow list is generated correctly
-			stateDB := extstatetest.NewTestStateDB(t)
+			stateDB := precompiletest.NewTestStateDB(t, map[common.Address][][]byte{})
```

Comment:
rather than exposing `NewTestStateDB`, I think we should do:
```golang
statedb, err := state.New(common.Hash{}, state.NewDatabase(rawdb.NewMemoryDatabase()), nil)
require.NoError(t, err)
wrappedStateDB := extstate.New(statedb)
```
here.